spec/system/ucas_matching/upload_matching_data_to_ucas_spec.rb

```diff
@@ -79,7 +79,6 @@ def then_the_file_has_been_uploaded
     expect(@latest_exception).to be_nil
 
     expect(WebMock).to have_requested(:post, 'https://transfer.ucasenvironments.com/api/v1/folders/685520099/files')
```

Comment:
Perhaps test the file can be unzipped here?

Reply:
No that doesn't work. See the comment in the PR for the background.

Reply:
@duncanjbrown made it work! https://github.com/DFE-Digital/apply-for-teacher-training/pull/2484/commits/3228e74c3f9d0146a03ee2ffae5b871e7643dc3b